rfcs/table.md

```diff
@@ -0,0 +1,321 @@
+# Table Component
+
+Represents control to render tabular data.
+
+## Features
+- Sorting
+- Basic mobile responsiveness (overflow scrollbar)
+
+
+## Unsupported
```

Comment:
Is this a list of *Unsupported Currently* but with plans to or *We do not plan on supporting* ?

Reply:
I would change this to _Unsupported Currently_ because I don't want to overpromise but underdeliver any features. Some features like Fixed Header and  Fixed Column which might be tricky to implement and my team doesn't need this.